# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/helix/hotfix-deadlink.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

See https://github.com/helix-editor/helix/issues/9391.

- NoTag <notag@t2sde.org>

--- helix-23.10/languages.toml.vanilla	2024-03-05 10:38:24.253023475 +0100
+++ helix-23.10/languages.toml	2024-03-05 10:38:44.452388205 +0100
@@ -2998,7 +2998,7 @@
 
 [[grammar]]
 name = "gemini"
-source = { git = "https://git.sr.ht/~sfr/tree-sitter-gemini", rev = "3cc5e4bdf572d5df4277fc2e54d6299bd59a54b3" }
+source = { git = "https://git.sr.ht/~nbsp/tree-sitter-gemini", rev = "3cc5e4bdf572d5df4277fc2e54d6299bd59a54b3" }
 
 [[language]]
 name = "templ"
